content/en/getting_started/synthetics/private_location.md

```diff
@@ -38,7 +38,7 @@ Your private locations test results display identically to your managed location
 
 ## Create your private location
 
-1. In the Datadog site, hover over **[UX Monitoring][5]** and select **Settings** > **Private Locations**. 
+1. In the Datadog site, hover over **[Digital Experience][5]** and select **Settings** > **Private Locations**. 
```

Comment:
```suggestion
1. In the Datadog site, hover over **[Digital Experience][5]** and select **Settings** > **Private Locations**. 
```
I think this link needs updated as it takes the user to https://app.datadoghq.com/synthetics/tests?query=&from_ts=1712093635145&to_ts=1712097235145&live=true and it should go to https://app.datadoghq.com/synthetics/settings/private-locations

Reply:
good catch, thanks!